models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md

```diff
@@ -0,0 +1,55 @@
+# unet-camvid-onnx-0001
+
+## Use Case and High-Level Description
+
+This is the UNet model that is designed to perform semantic segmentation. The model has been trained on the CamVid dataset from scratch using PyTorch framework. Training used median frequency balancing for class weighing. For details about the original floating point model, check out the [paper](https://arxiv.org/pdf/1505.04597.pdf).
+
+The model input is a blob that consists of a single image of "1x3x368x480" in BGR order. The pixel values are integers in the [0, 255] range.
+
+The model output for `unet-camvid-onnx-0001` is the per-pixel probabilities of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
+| Source framework  | PyTorch\*             |
```

Comment:
Is mParams and FLOPs values available for this model?

Reply:
ok, added